airgun/entities/contenthost.py

```diff
@@ -55,12 +56,39 @@ def execute_package_action(self, entity_name, action_type, value):
         view.progressbar.wait_for_result()
         return view.read()
 
+    def execute_module_stream_action(self, entity_name, action_type, module_name, stream_version):
+        """Execute remote module_stream action on a content
+
+        :param entity_name: content host name to remotely execute package
+            action on
+        :param action_type: remote action to execute. Can be one of 5: 'Enable', 'Disable'
+            'Install', 'Update', 'Remove', 'Reset' and 'Customize'
+        :param module_name: Module Stream name to remotely
+            install/upgrade/remove (depending on `action_type`)
+        :param stream_version: this uniduely identifies the module with Stream Version
+
+        :return: Returns a dict containing job status details
+        """
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        view.module_streams.search('name = {} and stream = {}'.format(module_name, stream_version))
+        view.module_streams.table.row(
+            name=module_name, stream=stream_version)['Actions'].fill(action_type)
+        view = JobInvocationStatusView(view.browser)
```

Comment:
Are all the actions invoke JobInvocationStatusView?

Reply:
Yes only customize options takes to another view. I'm not adding customize as this point in time.